pkg/supportbundle/defaultspec/spec.yaml

```diff
@@ -20,6 +20,18 @@ spec:
         selector:
           - app=kotsadm-postgres
         timeout: 10s
+    - exec:
+        collectorName: kotsadm-rqlite-db
+        name: kots/admin_console
+        containerName: rqlite
+        selector:
+          - app=kotsadm-rqlite
+        command:
+          - wget
+        args:
+          - -qO-
+          - kotsadm:password@localhost:4001/db/backup?fmt=sql
```

Comment:
the password is NOT `password` in production environments, only in dev environments. the password is saved in a secret named `kotsadm-rqlite`